dss/operations/secrets.py

```diff
@@ -0,0 +1,312 @@
+                   "--json": dict(
+                       default=False,
+                       action="store_true",
+                       help="format the output as JSON if this flag is present")})
+def list_secrets(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Print a list of names of every secret variable in the secrets manager
+    for the given store and stage
+    """
+    store_name, stage_name = get_secretsmanager_names(args)
+    arn_prefix, store_prefix = get_secretsmanager_prefixes(args)
+
+    paginator = secretsmanager.get_paginator('list_secrets')
+
+    secret_names = []
+    for response in paginator.paginate():
+        for secret in response['SecretList']:
+
+            # Get resource IDs
+            secret_name = secret['ARN']
+            short_secret_name, long_secret_name = long_short_resource_names(secret_name, arn_prefix, store_prefix)
+
+            # Only save secrets for this store and stage
+            if short_secret_name.startswith(store_prefix):
+                if args.long:
+                    secret_names.append(long_secret_name)
+                else:
+                    secret_names.append(short_secret_name)
+
+    secret_names.sort()
+
+    if args.json:
+        print(json.dumps(secret_names))
+    else:
+        for secret_name in secret_names:
+            print(secret_name)
+
+
+@events.action("get",
+               arguments={
+                   "--stage": dict(
+                       required=False,
+                       help="the stage for which secrets should be listed"),
+                   "--secret-name": dict(
+                       required=True,
+                       nargs="*",
+                       help="name of secret or secrets to retrieve (list values can be separated by a space)"),
+                   "--long": dict(
+                       default=False,
+                       action="store_true",
+                       help="use long (full ARN) identifiers when printing secret variable(s)"),
+                   "--json": dict(
+                       default=False,
+                       action="store_true",
+                       help="format the output as JSON if this flag is present")})
+def get_secret(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Get the value of the secret variable (or list of variables) specified by
+    the --secret-name flag
+    """
+    # Note: this function should not print anything except the final JSON,
+    # in case the user pipes the JSON output of this script to something else
+    store_name, stage_name = get_secretsmanager_names(args)
+    arn_prefix, store_prefix = get_secretsmanager_prefixes(args)
+
+    # Make sure a secret name was specified
+    if args.secret_name is None or len(args.secret_name) == 0:
+        logger.error("Unable to set secret: no secret name was specified! Use the --secret-name flag.")
+        sys.exit()
```

Comment:
Consider using `RuntimeError` instead of `sys.exit()`:
```
raise RuntimeError("Unable to set secret: no secret name was specified! Use the --secret-name flag.")
```

Reply:
Okay, done!